### ISSUE TITLE
[Logging v2] Installed App with one option true in additionalLoggingSources shows false for all when checking Values yaml view

### ISSUE THREAD

**Issue by izaac** [rancher/dashboard#1316]
After installing `rancher-logging` in one of the supported Control Plane Additional sources I don't see the values used during installation properly.

Sample from a k3s openrc using 

```
   k3s:
    container_engine: openrc
    enabled: true
```

Checking the Installed App shows:

```
additionalLoggingSources:
  eks:
    enabled: false
  k3s:
    container_engine: systemd
    enabled: false
  rke:
    enabled: false
  rke2:
    enabled: false
```

![Screen Shot 2020-09-19 at 12 28 55 PM](https://user-images.githubusercontent.com/27659/93687658-20b36700-fa74-11ea-9c01-12c6fffa66c4.png)

_** The app was installed properly with the control plane option enabled **_
rancher/dashboard#1109

Rancher version `master-head (09/19/2020)` _bd88c23_ HA k8s

**Comment by codyrancher** [comment]
@izaac There doesn't seem to be a way to navigate to the installed apps in the latest. So is this still an issue?

If this is still an issue then I'd like @vincent99 to chime in because I don't think this was caused by the logging chart pages.

It looks like the yaml is referencing spec.charts.values [here](https://github.com/rancher/dashboard/blob/b56af4e94ad0df019ef72cd031cefd9335433e38/detail/catalog.cattle.io.release.vue#L37)

If I look at the api I see two `values` objects, one at `spec.charts.values` and one at `spec.values`. The `spec.values` appears to be what was set when installed. Is is the yaml incorrect?
![image.png](https://zube.io/files/rancher/2a1db2597ac9f07cbc9c84f8837abc05-image.png)

![image.png](https://zube.io/files/rancher/b247b76ffa6b0ec53fed6435eb8f1af4-image.png)

**Comment by vincent99** [comment]
TL;DR: The `Values YAML` tab of detail is showing the wrong info.

`spec.chart.values` is the entire original default values.yaml from the definition of the chart it was deployed with. (this is why in your screenshot it's long and has tons of fields)

`spec.values` is the user's input.  They are shown the full chart values (or a form equivalent) when launching the app, and edit whatever parts they want.  The dashboard then does a diff of the original vs the user's desired values and saves just that *diff* of the two as `spec.values`.  (this is why in your screenshot it's just the enabled flag basically, plus the special global fields we silently add).

So on the detail page the values that are shown should be the reconstituted combination of the two (which _is_ what you get if you try to upgrade), but is currently showing just the original values and ignoring the user values.

**Comment by izaac** [comment]
Rancher version `master-head (09/23/2020)` _e9a98a3_
`spec.values` is now showing the user entered `additionalLoggingSources` from Installed Apps
